packages/react/src/components/CTA/CTA.js

```diff
@@ -68,7 +72,7 @@ const renderCTA = ({ style, type, ...cta }) => {
           href={cta.href}
           target={_external(type)}
           onClick={e => _jump(e, type)}>
-          {cta.copy}
+          {cta.copy && <span>{cta.copy}</span>}
```

Comment:
@photodow Don't think this needs to be part of this review. Will make note to address for that component.

Reply:
I might push back on this one slightly. The addition of the `span` in this pull request was intentional. I think we should consider proper placement, or alternative to including it.

Reply:
@photodow actually because the `cta.copy` has to be there, we can revert the changes as I made them for the case where it would only be an icon link where the icon had left margin applied to it